mne_bids/read.py

```diff
@@ -147,9 +147,7 @@ def _read_events(events_data, event_id, raw, verbose=None):
         verbose=verbose
     )
     all_dur = raw.annotations.duration
-    if all_events.size == 0 and verbose:
-        # Confirm from filename dataset is not likely resting state
-        if '_task-rest' not in raw.filenames[0]:
+if all_events.size == 0 and `_task_rest` not in raw.filenames[0]:
             warn('No events found or provided. Please add annotations '
```

Comment:
Indentation is off now, sorry ;)

Reply:
:) 
Thanks - my bad I didn't check the fix above thoroughly enough
Just re-added `verbose == True`
(This is providing that we do indeed want the warning again (not logger.info))